src/components/Button/Button.st.css

```diff
@@ -355,3 +358,55 @@
 .root:upgrade:size(large):mobile, .root:priority(primary):upgrade:size(large):mobile, .root:priority(basic):upgrade:size(large):mobile{
     padding: calculate(string(-), 17px, value(secondaryBorderWidth)) 16px;
 }
+
+.root .affixIcon svg path {
+    fill: none;
+    transition: stroke .2s ease-in-out;
```

Comment:
Couldn't we just add the `stroke: value(PrimaryTextColor);` here and remove the:

```
.root .affixIcon svg path,
.root:priority(basic) .affixIcon svg path,
.root:priority(primary) .affixIcon svg path {
    stroke: value(PrimaryTextColor);
}

```